### DISCUSSION TITLE
NoT SYNCED

### DISCUSSION THREAD

**Discussion by alvaropietro** [Chia-Network/chia-blockchain#3397] [Support]
First, after upgrading to a new version 1.1.3 it synchronized very fast, after putting some plots to start, it appeared as not synchronized. And it does not synchronize released ports 8444 at all. Any tips to make it work?

**Comment by shanduur** [comment]
Same issue here,

**Reply by caelumdev** [reply]
+1

**Reply by morovid95** [reply]
I had the same issue. First i checked my Bios and my System time it need to be the same! after this it doesn't work and i deleted my .chia directory and restarted chia after this it goes syncing very well for me (sry for my English)

**Reply by eppunay** [reply]
Currently having the same issue right now stuck at block height 225030.

**Reply by iwwxxl** [reply]
same issue +1

**Reply by tobiasncorso** [reply]
same issue +1

**Comment by wangpr0606** [comment]
+1

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
